stdlib/_typeshed/__init__.pyi

```diff
@@ -312,3 +312,6 @@ TraceFunction: TypeAlias = Callable[[FrameType, str, Any], TraceFunction | None]
 #   https://github.com/microsoft/pyright/issues/4339
 class DataclassInstance(Protocol):
     __dataclass_fields__: ClassVar[dict[str, Field[Any]]]
+
+# Any thing that can be passed to the int constructor
+_AcceptedByInt: TypeAlias = str | ReadableBuffer | SupportsInt | SupportsIndex | SupportsTrunc  # noqa: Y047
```

Comment:
Nit: the name of this whole module is private, and I think that gives enough of a "Hey! These types aren't available at runtime!" warning for users. I'd prefer to give this alias a public name, since it is, after all, meant to be "publicly exported" from this private module 😀

```suggestion
AcceptedByInt: TypeAlias = str | ReadableBuffer | SupportsInt | SupportsIndex | SupportsTrunc  # noqa: Y047
```